extensions/wikia/GlobalShortcuts/scripts/GlobalShortcutsHelp.js

```diff
@@ -19,7 +19,9 @@ define('GlobalShortcutsHelp',
 					Wikia.getMultiTypePackage({
 						mustache: 'extensions/wikia/GlobalShortcuts/templates/KeyCombination.mustache,' +
 							'extensions/wikia/GlobalShortcuts/templates/GlobalShortcutsController_help.mustache',
+						messages: 'GlobalShortcuts,TemplateClassificationGlobalShortcuts',
```

Comment:
If this is just for `template-class-global-shortcuts-press-to-explore-shortcuts`, I think it should be shortened to  `global-shortcuts-press-to-explore-shortcuts`  and defined in the `GlobalShortcuts` messages package rather than having this cross-dependency with the TemplateClassification extension where the `TemplateClassificationGlobalShortcuts` package is defined.